src/server/routes/__tests__/submissions.unit.spec.js

```diff
@@ -673,7 +699,61 @@ describe('submissions.unit', () => {
       Submission.findByPk.mockRejectedValueOnce(error)
 
       const result = await patchSubmissionByIdHandler(
-        getSubmissionRequest({ status: 'COMPLETE' }),
+        getSubmissionRequest({ status: 'SUBMITTED' }),
+        getMockResponseToolkit()
+      )
+
+      expect(result).toBe(SERVER_ERROR_SYMBOL)
+    })
+
+    it('should log an error but still return 200 when the call to update an activity in CRM returns an ErrorMessage', async () => {
+      Submission.findByPk.mockResolvedValueOnce(getFoundSubmission())
+      updateActivityCRM.mockResolvedValue({
+        '@odata.context':
+          'https://dynamics.com/api/data/v9.1/defra_CreateRCRActivityResponse',
+        RCRActivityId: null,
+        ReturnStatus: 'error',
+        SuccessMessage: '',
+        ErrorMessage: 'Failed to update activity'
+      })
+
+      const result = await patchSubmissionByIdHandler(
+        getSubmissionRequest({ status: 'SUBMITTED' }),
+        getMockResponseToolkit()
+      )
+
+      expect(result.statusCode).toBe(200)
+      expect(logger.error).toHaveBeenCalledWith(
+        'failed to update activity in CRM for contact-identifier-111',
+        'Failed to update activity'
+      )
```

Comment:
One expect statement per test, please - think this could easily be refactored to a describe statement, with the setup done in a `beforeEach`